gold/nanomips-insn.def

```diff
@@ -235,6 +241,8 @@ NTT(PCREL16,          RELS(R(PC25_S1)),                               INSNS(BALC
 NTT(PCREL_NMF,        RELS(R(PC25_S1)),                               INSNS(LAPC48, JALRC16(SREG)))
 NTT(PCREL16_LONG,     RELS(R(PC25_S1)),                               INSNS(ALUIPC32(PC_HI20, SREG), ORI32(LO12, SREG, SREG), JALRC16(SREG)))
 NTT(PCREL32_LONG,     RELS(R(PC25_S1)),                               INSNS(ALUIPC32(PC_HI20, SREG), ORI32(LO12, SREG, SREG), JALRC32))
+NTT(BALC_TRAMP,       RELS(R(PC25_S1)),                               INSNS(BALC16_FIXED, BC16_FIXED, BC32))
+NTT(BALC_CALL,        RELS(R(PC25_S1)),                               INSNS(BALC16))
```

Comment:
I don't see where the BALC_CALL transformation type is actually affected by changing the  the call-site to go to the trampoline instead of the final function. You can't just be changing the BALC32 to BALC16, because the next expansion phase will just expand it right back to BALC32 - this is indeed what seems to be happening looking at the `--debug=target' log output. You need to change the relocation symbol as well.